from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+from estimators import bandits
+from estimators.bandits import ips
+
+def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict:
+    if len(bucket) == 0 or config == {}:
+        return {}
+    else:
+        e = bandits.ips.Estimator()
```

Comment:
Great start!
Now, since we are able to consume estimators from the package, we need to do the following:
instead of hard-coded processing of (0.5, 1, 0.5) by ips estimator we need to process data from input data frame by estimators specified in the config.

Reply:
Should this do it? I'm guessing for multiple values this might be some kind of loop (but not sure)? Thanks!

Reply:
Sure, it should be multiple loops:
- for every row in data frame
- for every policy from the config
- for every estimator for a given policy

Reply:
still working on the last estimator bit; this is supposed to print out `ips`, `snips`, etc?

Reply:
Yes, config is specifying which estimators (ips/snips/mle/...) we should use for certain policies.

Reply:
think I got it to print the ips and ran it through one with two and printed out both. Feedback welcome.